src/Parser/Mage/Frost/Modules/Features/ThermalVoid.js

```diff
@@ -14,29 +14,38 @@ class ThermalVoid extends Module {
 	}
 
   casts = 0;
+  buffApplied = 0;
+  extraUptime = 0;
 
   on_initialized() {
 	   this.active = this.combatants.selected.hasTalent(SPELLS.THERMAL_VOID_TALENT.id);
   }
 
   on_toPlayer_applybuff(event) {
-    const spellId = event.ability.guid;
-    if(spellId === SPELLS.ICY_VEINS.id && event.prepull) { // catch prepull cast by buff being present on pull
+    if(event.ability.guid === SPELLS.ICY_VEINS.id && event.prepull) { // catch prepull cast by buff being present on pull
```

Comment:
Ill fix those. I removed them because it seemed cleaner to not split it out like that and the single line without the constant is what i did in my other modules, so i was changing this one to match those. But ill change it back if thats the preferred way to do it ... and then ill change my other modules back the next time i modify them so they are standard

Reply:
Thanks. I know for some of these smaller modules it seems potentially like an unneeded step, but just to keep the style consistent I would update. And that is fine for the other stuff, just slowly convert it over.